src/components/Asset/AssetActions/Compute/index.tsx

```diff
@@ -406,21 +369,32 @@ export default function Compute({
     <>
       <div className={styles.info}>
         <FileIcon file={file} isLoading={fileIsLoading} small />
-        <Price accessDetails={asset?.accessDetails} conversion />
+        {isPriceTypeSupported ? (
+          <Price accessDetails={asset?.accessDetails} conversion />
+        ) : (
+          <Alert
+            text={`Dynamic pricing with pools [is deprecated](https://blog.oceanprotocol.com/ocean-market-changes-3384fd7e113c).`}
+            state="info"
+          />
+        )}
       </div>
 
       {asset.metadata.type === 'algorithm' ? (
         <>
           <Alert
-            text="This algorithm has been set to private by the publisher and can't be downloaded. You can run it against any allowed data sets though!"
+            text={
+              "This algorithm has been set to private by the publisher and can't be downloaded. You can run it against any allowed data sets though!"
```

Comment:
this is something we need to solve as this goes into the compute to data design. We need a new mechanism to signal algos to be downloadable or not.

Reply:
whoops, ignore, mixed something up so there's no issue here, push incoming